bucket/cryptomator.json

```diff
@@ -21,5 +21,10 @@
         "hash": {
             "url": "https://dl.bintray.com/cryptomator/cryptomator/$version/Cryptomator-$version-x64.exe.sha256"
         }
-    }
+    },
+    "persist": "data",
+    "post_install": [
+        "# change appdata settings folder",
+        "(Get-Content \"$dir\\app\\Cryptomator.cfg\").replace('~/AppData/Roaming/Cryptomator', '../data').replace('\\', '/') | Set-Content \"$dir\\app\\Cryptomator.cfg\""
```

Comment:
- Why to replace `\` with `/`?
- No encoding
    - Use `ASCII`

Reply:
I really don't see why replace \ with /.
Removed that part and added ASCII encoding